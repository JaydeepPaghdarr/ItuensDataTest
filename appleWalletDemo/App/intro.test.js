import React from 'react';
import {TextInput} from 'react-native';
import renderer from 'react-test-renderer';

// jest.setTimeout(15000);

test('renders the TextInput component', () => {
  const tree = renderer
    .create(<TextInput autoCorrect={false} value="apple" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
