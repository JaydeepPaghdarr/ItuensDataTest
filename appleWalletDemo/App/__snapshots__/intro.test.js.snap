// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the TextInput component 1`] = `
<TextInput
  allowFontScaling={true}
  autoCorrect={false}
  rejectResponderTermination={true}
  underlineColorAndroid="transparent"
  value="apple"
/>
`;
